mods/d2k/weapons/other.yaml

```diff
@@ -60,11 +60,11 @@ WormJaw:
 	ReloadDelay: 10
 	InvalidTargets: Structure, Infantry
 	Range: 1c512
-	Warhead@1Dam: SpreadDamage
+	Warhead@1Dam: SpreadDamage # HACK: The warhead is needed for targeting
 		InvalidTargets: Structure, Infantry
-		Spread: 768
-		Falloff: 100, 100, 0
-		Damage: 10000
+		Spread: 0
+		Falloff: 0,0
+		Damage: 0
```

Comment:
Removed it. If we want to wait for #12605 is up to the reviewers. (I'm fine with either.)